lib/mods/replace-window-functions.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  formatReplacement,
+  getOriginal,
+} = require('../originals-helper.js');
+
+/**
+ * @param {*} node - the node to examine
+ *
+ * @return {boolean} whether the node represents a window function
+ */
+function isWindowProperty(node) {
+  const symbol = node.getExpression().getSymbol();
+  const firstDeclaration = (symbol) ? symbol.getDeclarations()[0] : undefined;
+
+  // TODO: may require a deeper check
```

Comment:
This should be taken out before merge, but I wanted to highlight the fact that this could require looking deeper - I'm not sure what test would show this though